### PR TITLE
Hw exception

### PR DESCRIPTION
**Pull request by KulAndriy**

### CODE REVIEW

File: src/main/java/exception/TestExc.java [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+import java.util.Iterator;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class TestExc {
+    /**
+     * TASK #1
+     * Витягнути ліст імен, Видалити всіх Олегів з ліста (відповідно, щоб у лісті було їх хоча б кілька)
+     */
+    public List<String> removeEmployee(List<Employee> employeeList, ExecutionType executionType, String targetName) throws NoSuchEmployeeException {
```
Comment (aromanyshyn-cartera) [inline src/main/java/exception/TestExc.java:16]: NoSuchEmployeeException -  is Runtime Exception. So there is no need to add it after method signature

File: src/main/java/main/EmployeesProcessor.java [hunk 2]
```diff
@@ -185,33 +190,37 @@ public boolean isEmployeeLivesByAddress(List<Employee> employeeList, ExecutionTy
...
                 break;
             }
             case STREAM:{
                 lastNameOfEmployeesWithConcatenation = employeeList.stream()
-                        .filter(s->s.getAge()>70)
+                        .filter(s->s.getAge()>age)
                         .map(s->s.getLastName().concat(concatedStringToLastName))
                         .collect(Collectors.toList());
             }
+            if (lastNameOfEmployeesWithConcatenation.isEmpty()) throw new EmployeeCollectionIsEmptyException(exceptionMessage + age);
```
Comment (aromanyshyn-cartera) [inline src/main/java/main/EmployeesProcessor.java:223]: it's enough to do this verification only once before return

File: src/main/java/exception/TestExc.java (hunk 1 above)
Comment (AnnaRomanysh) [reply src/main/java/exception/TestExc.java:16]: NoSuchEmployeeException - is Runtime Exception. So there is no need to add it after method signature

Comment (AnnaRomanysh) [inline src/main/java/exception/TestExc.java]: NoSuchEmployeeException - is Runtime Exception. So there is no need to add it after method signature

File: src/main/java/main/EmployeesProcessor.java (hunk 2 above)
Comment (AnnaRomanysh) [inline src/main/java/main/EmployeesProcessor.java]: it's enough to do this verification only once before return

**Review by AnnaRomanysh — CHANGES_REQUESTED** [review]

**Review by AnnaRomanysh — APPROVED** [review]

**Pull request merged**